### ISSUE TITLE
Projects - Separate the projects-table container into a container + a view-only component

### ISSUE THREAD

**Issue by jamescd18** [Northeastern-Electric-Racing/PM-Dashboard-v2#137]
In order to reduce the complexity of each individual react component, we want to try and separate different functionality between view-only components and container components. Currently, the projects-table component was built before realizing that separation would be beneficial for this component. The goal of this issue is to separate the view functionality from the data fetching and transformation functionality.

Basically, we want to put the table configuration details in a view-only component that accepts the array of display projects as a component prop. Then the data fetching and transformation logic will stay in the projects-table container component.

**Comment by jamescd18** [comment]
- [ ] Understand how the change requests table is split into two components
  - the data fetching component
  - the view component that configures the display of the table
- [ ] Create the new view-only component
- [ ] move the view logic into the new component
